broadinstitute/wfl
# [GH-1139] Added new log namespace

### Purpose
We want to replace the current logging with json output so that it can be picked up by stackdriver and provide easier querying.

https://broadinstitute.atlassian.net/browse/GH-1139

### Changes
- Added new log namespace with macros for info, warn, debug and error.

### Review Instructions


- Build WFL and start the server locally
- Watch the stream of logs in the terminal and make sure they are all in json format with a severity, message and timestamp at the least
- Hit some of the endpoints via swagger and make sure that there are log entries coming through for those as well
- Try to produce an error to make sure that exceptions are logged as ERROR severity log entries
